aws/PAS-Component-Single-Deployment.json

```diff
@@ -2390,5 +2390,9 @@
             "Ref":"LogGroup"
          }
       }
+   },
+   "GetVaultMachineIp":{
+    "Description": "PrivateIP",
+    "Value" : { "Fn::GetAtt" : [ "PVWAMachine", "PrivateIp" ]}
    }
 }
```

Comment:
How is this related to azure? is it for the cross cloud feature?

Reply:
Removed